### ISSUE TITLE
Timeout during tutorial

### ISSUE THREAD

**Issue by kofalt** [flynn/flynn#1015]
Reproducable on my machine using the demo vagrant instance.

During the [tutorial](https://flynn.io/docs/using-flynn#release), on the second `git push flynn master`:

```text
-----> Creating release...
remote: Error deploying app release: Timed out waiting for deployment completion!
To ssh://git@demo.localflynn.com:2222/example.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'ssh://git@demo.localflynn.com:2222/example.git'
```

Full outout: https://gist.github.com/kofalt/cad8b88c9a8301bd07ec

Linux x64, flynn `20150209`, vagrant `1.6.3`, using flynn repo at 43acb43 and node-js-example at 9a70422

**Comment by kofalt** [comment]
Notably:

```
 $ flynn log flynn-7b33842c5339433e8dbcec4e1a07bf74
Listening on 8080
```

I would have expected to see the `I've made a change!` line added to this release.

A second try yields:

```text
-----> Creating release...
remote: Error deploying app release: validation_error: Cannot create deploy, there is already one in progress for this app.
To ssh://git@demo.localflynn.com:2222/example.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'ssh://git@demo.localflynn.com:2222/example.git'
```

**Comment by archSeer** [comment]
@kofalt I cannot reproduce it, can you `flynn-host upload-debug-info` the next time it happens? https://flynn.io/docs/development#upload-logs-and-system-information-to-a-gist

**Comment by kofalt** [comment]
I went ahead & started from a clean state, with `vagrant destroy -f && time vagrant up` then following the tutorial again.

Hoping into the VM with `vagrant ssh`, I ran your requested command:

```text
$ sudo flynn-host upload-debug-info
02:28:08.941531 upload-debug-info.go:48: error adding tmp-flynn-host.log: open /tmp/flynn-host.log: no such file or directory
02:28:10.642160 upload-debug-info.go:71: Debug information uploaded to: https://gist.github.com/bfdb21fa4bcecedd3c36
```

Link: https://gist.github.com/bfdb21fa4bcecedd3c36

**Comment by archSeer** [comment]
Seems like this run was successful though? Your logs show two successful deploys.

**Comment by archSeer** [comment]
From the discussion on IRC, we assume this was fixed on master with #1005, but the release images weren't updated yet. The fix should be available tonight :)

**Comment by kofalt** [comment]
Sounds good. I'll build HEAD tomorrow and try that :D

**Comment by lmars** [comment]
@kofalt the fix for this has now been released (it is in the `v20150217.0` release), let us know how you get on using that version.
